code/game/objects/items/storage/storage.dm

```diff
@@ -455,16 +454,16 @@ That's done by can_be_inserted(). Its checks are whether the item exists, is an
 The stop_warning parameter will stop the insertion message from being displayed. It is intended for cases where you are inserting multiple
 items at once, such as when picking up all the items on a tile with one click.
 user can be null, it refers to the potential mob doing the insertion.**/
-/obj/item/storage/proc/handle_item_insertion(obj/item/W, prevent_warning = 0, mob/user)
-	if(!istype(W))
+/obj/item/storage/proc/handle_item_insertion(obj/item/Warn, prevent_warning = 0, mob/user)
```

Comment:
Three things:
- `Warn` makes no sense to be the variable name; this is kinda more confusing than just a random letter like it used to be.
- We use snake_case; not camelCase or PascalCase. Your variable here is PascalCase. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case
- Use `FALSE` instead of `0` for booleans.

So something more like `obj/item/new_item` would be more appropriate.

Reply:
replace Warn like you proposed.
and replace = 0 by = FALSE like you proposed